packages/notifications/demo/global-alert.stories.mdx

```diff
@@ -0,0 +1,53 @@
+import { Meta, ArgsTable, Canvas, Story } from '@storybook/addon-docs';
+import { GlobalAlert, Close, Paragraph, Title } from '@zendeskgarden/react-notifications';
+import { GlobalAlertStory } from './stories/GlobalAlertStory';
+
+<Meta
+  title="Packages/Notifications/GlobalAlert"
+  component={GlobalAlert}
+  subcomponents={{
+    'GlobalAlert.Anchor': GlobalAlert.Anchor,
+    'GlobalAlert.Button': GlobalAlert.Button,
+    'GlobalAlert.Close': GlobalAlert.Close,
+    'GlobalAlert.Content': GlobalAlert.Content,
+    'GlobalAlert.Title': GlobalAlert.Title
+  }}
+/>
+
+# API
+
+<ArgsTable />
+
+# Demo
+
+<Canvas>
+  <Story
+    name="GlobalAlert"
+    args={{
+      type: 'info'
+    }}
+    argTypes={{
+      anchor: { name: 'children', type: 'string', table: { category: 'Anchor' } },
+      isExternal: { control: { type: 'boolean' }, table: { category: 'Anchor' } },
+      button: { name: 'children', type: 'string', table: { category: 'Button' } },
+      isBasic: { control: { type: 'boolean' }, table: { category: 'Button' } },
+      ariaLabel: { name: 'aria-label', type: 'string', table: { category: 'Close' } },
```

Comment:
Please add default `args` text for the `aria-label`. See `Modal` `Close`, `DrawerModal.Close` and `TooltipModal.Close` for details. Doing so bypasses the dev warning out of the box, and shows we are using the component as intended.